internal/usage/usage.go

```diff
@@ -403,33 +403,33 @@ dbName and collection are required only for built-in roles.`
 	OplogMinRetentionHours                    = "Minimum retention window for cluster's oplog expressed in hours."
 	OplogSizeMB                               = "Storage limit of cluster's oplog expressed in megabytes."
 	SampleRefreshIntervalBIConnector          = "Interval in seconds at which the mongosqld process re-samples data to create its relational schema."
 	SampleSizeBIConnector                     = "Number of documents per database to sample when gathering schema information."
 	ExporterClusterName                       = "One or more comma separated cluster names to import"
-	OperatorIncludeSecrets                    = "Generate kubernetes secrets with data for projects, users, deployments entities" //nolint:gosec //This is just a message, not a credential
+	OperatorIncludeSecrets                    = "Flag that generates kubernetes secrets with data for projects, users, deployments entities."
 	OperatorTargetNamespace                   = "Namespaces to use for generated kubernetes entities"
 	OperatorVersion                           = "Version of Atlas Kubernetes Operator to generate resources for."
-	OperatorVersionInstall                    = "Version of the operator to be installed."
+	OperatorVersionInstall                    = "Version of the operator to install."
 	OperatorTargetNamespaceInstall            = "Namespace where to install the operator."
-	OperatorWatchNamespace                    = "List of namespaces which the operator will be listen to."
+	OperatorWatchNamespace                    = "List that contains namespaces that the operator will listen to."
 	OperatorProjectName                       = "Name of the project to create or use with the installed operator."
-	OperatorImport                            = "Flag to import existing Atlas resources into the cluster to be managed by the operator."
-	OperatorAtlasGov                          = "Flag to configure Atlas for Government as target of the operator."
+	OperatorImport                            = "Flag that indicates whether to import existing Atlas resources into the cluster for the operator to manage."
+	OperatorAtlasGov                          = "Flag that indicates whether to configure Atlas for Government as a target of the operator."
 	KubernetesClusterConfig                   = "Path to the kubeconfig file to use for CLI requests."
-	KubernetesClusterContext                  = "The name of the kubeconfig context to use."
+	KubernetesClusterContext                  = "Name of the kubeconfig context to use."
 	ExportID                                  = "Unique string that identifies the AWS S3 bucket to which you export your snapshots."
-	RequiredRole                              = "To use this command, you must authenticate with a user account or an API key that has the %s role."
+	RequiredRole                              = "Role required to use this command. You must authenticate with a user account or an API key that has the %s role."
```

Comment:
@dianchenghu and @blva, sorry, since this isn't a typical flag description, I think we should change this back.